packages/viewer/src/lib/transforms/transformation-delta.ts

```diff
@@ -23,10 +24,14 @@ export function chooseOrthogonalVector(
 }
 
 /**
- * Computes the rotation matrix for two normal vectors.
+ * Computes the rotation matrix for two normal vectors. If both normals are anti-parallel,
```

Comment:
nit: Looks like there's a duplicate reference to `anti-parallel` here, could we change this to `If both normals are neither parallel or anti-parallel`?